index.html

```diff
@@ -126,7 +127,35 @@
+                                <ol>
+                                        <li>Dostępne są dwa query buildery: użycie <a href="https://github.com/laravel/framework/blob/v5.0.32/src/Illuminate/Database/Eloquent/Builder.php">pierwszego z nich</a> powoduje zwrócenie obiektów Active Record, natomiast <a href="https://github.com/laravel/framework/blob/v5.0.32/src/Illuminate/Database/Query/Builder.php">drugi</a> zwraca tablice.</li>
+                                        <li>Eloquent\Builder nie pozwala na filtrowanie wyników zapytania po wartościach z JOINowanych tabel i zwrócenie instancji modelu jednocześnie. <a href="https://www.facebook.com/groups/laravelpolska/permalink/1587069624864733/">Dyskusja</a>.</li>
+                                        <li>Wołając na obiekcie AR metodę <a href="https://github.com/laravel/framework/blob/v5.0.32/src/Illuminate/Database/Eloquent/Model.php#L1460">save()</a> zwrócona wartość pozwoli nam ustalić czy operacja się powiodła. Używając metody <a href="https://github.com/laravel/framework/blob/v5.0.32/src/Illuminate/Database/Eloquent/Model.php#L525">create()</a> powyższej informacji dostaczy nam <a href="https://github.com/laravel/framework/blob/v5.0.32/src/Illuminate/Database/Eloquent/Model.php#L175">publiczna właściwość klasy modelu</a>. W żadnym z powyższych wypadków nie zostanie automatycznie przerwana transakcja. <a href="https://www.facebook.com/groups/223889134481096/permalink/336420516561290/">Dyskusja</a>.</li>
+                                </ol>
+                                <p>Dostęp statyczny:</p>
+                                <ol>
+                                		<li>W Laravelu dostępny jest moduł Dependency Injection oparty o Reflection i <a href="http://laravel.com/docs/5.1/container#resolving">mogący działać bez żadnej konfiguracji</a>. Znajdowały się w nim pewne <a href="https://github.com/laravel/framework/pull/4824">niedoskonałości</a> mogące skutecznie utrudnić debugowanie.</li>
+                                		<li>Alternatywą dla Dependency Injection są <a href="http://laravel.com/docs/5.1/facades">Fasady</a>. Ich nadużywanie prowadzi do sytuacji, w której trudno jest określić zależności danej klasy. Planowane jest wprowadzenie funkcjonalności pozwalającej na <a href="http://laravel.com/docs/master/blade#service-injection">dostęp do serwisów z poziomu template'ów.</a></li>
+                                        <li>Ze względu na Fasady w <a href="https://github.com/laravel/framework/blob/v5.0.32/src/Illuminate/Support/Facades/Facade.php#L42">produkcyjnym kodzie</a> znajdują się metody służące wyłącznie testowaniu.</li>
+                                        <li>Zapisywanie obiektu do bazy realizowane jest za pomocą <a href="https://github.com/laravel/framework/blob/v5.0.32/src/Illuminate/Database/Eloquent/Model.php#L1185">statycznej metody save()</a>, której mockowanie w testach jednostkowych jest mocno utrudnione.</li>
```

Comment:
Norbercie, warstwa abstrakcji (czy raczej to co sie za nia kryje) tez musi mieć jakieś testy. Prędzej czy później dojdziesz do momentu, w którym zawolasz persist() albo save(). Oczywiście - może być to test funkcjonalny (tak by pewnie sugerowała Clean Architecture). 
Uważam wszakże, ze pisanie **zawsze** kodu doskonale oderwanego od frameworka, jest działaniem niepotrzebnym. W pewnych sytuacjach wręcz niemożliwym (sam wspominales o CA i Laravelu w ostatni poniedziałek w Gdańsku)
W imię zasad to Franz Maurer kolegę zastrzelił i nic dobrego z tego nie wyniknelo ;)


-- 
Pozdrawiam/Regards,
Iwan.